File: code/ec126.pas
program Bai126;
var a, b, c,d: integer;
procedure Hoan_doi(var x, y:integer);
var tam : integer; 
begin
    tam := x;
    x := y;
    y := tam;
end;
begin
    readln(a, b, c,d);
    if a > b then Hoan_doi(a,b);
    if b > c then Hoan_doi(b,c);
    if c > d then Hoan_doi(c,d);
    if a > b then Hoan_doi(a,b);
    if b > c then Hoan_doi(b,c);
    if c > d then Hoan_doi(c,d);
    if ((a=b) and (a<>c) and (c<>d)) or  ((b=c) and (a<>b) and (c<>d)) 
    or ((c=d) and (c<>b) and (b<>a)) then writeln('YES')
    else writeln('NO');
end.

File: code/ec126.py
if a > b:
  a,b = b,a
if b > c:
  b,c = c,b
if c> d:
  c,d = d,c
if a> b:
  a,b = b,a
if b > c:
  b,c = c, b
if c> d:
   c,d = d, c
if ((a == b) and (b !=c) and (c != d)) or ((a != b) and (b == c) and (c != d)) or ((a != b) and (b != c) and (c == d)):
  print("YES")
  quit
else:
  print('NO')